python/feldera/rest/feldera_client.py

```diff
@@ -200,35 +200,46 @@ def start_pipeline(self, pipeline_name: str):
         )
 
         while True:
-            status = self.get_pipeline(pipeline_name).deployment_status
+            resp = self.get_pipeline(pipeline_name)
+            status = resp.deployment_status
 
             if status == "Running":
                 break
             elif status == "Failed":
-                raise RuntimeError("Failed to start pipeline")
+                raise RuntimeError(
+                    "error: cannot START a failed pipeline\n"

```

Comment:
Here and in other places in this PR, let's make the error message more explicit:

```
Unable to START the pipeline.
Reason: The pipeline is in a FAILED state due to the following error:
'<error_message>'
```